src/linear_algebra/prod.lean

```diff
@@ -234,6 +236,22 @@ begin
   rw [←prod_map_comap_prod, submodule.prod_bot],
 end
 
+@[simp]
+lemma prod_map_id : (id : M →ₗ[R] M).prod_map  (id : M₂ →ₗ[R] M₂) = id :=
+linear_map.ext $ λ _, prod.mk.eta
+
+@[simp]
+lemma prod_map_one : (1 : M →ₗ[R] M).prod_map  (1 : M₂ →ₗ[R] M₂) = 1 :=
+linear_map.ext $ λ _, prod.mk.eta
+
+lemma prod_map_comp
+  (f₁₂ : M →ₗ[R] M₂) (f₂₃ : M₂ →ₗ[R] M₃) (g₁₂ : M₄ →ₗ[R] M₅) (g₂₃ : M₅ →ₗ[R] M₆) :
+  f₂₃.prod_map  g₂₃ ∘ₗ f₁₂.prod_map  g₁₂ = (f₂₃ ∘ₗ f₁₂).prod_map  (g₂₃ ∘ₗ g₁₂) := rfl
+
+lemma prod_map_mul
+  (f₁₂ : M →ₗ[R] M) (f₂₃ : M →ₗ[R] M) (g₁₂ : M₂ →ₗ[R] M₂) (g₂₃ : M₂ →ₗ[R] M₂) :
+  f₂₃.prod_map  g₂₃ * f₁₂.prod_map  g₁₂ = (f₂₃ * f₁₂).prod_map  (g₂₃ * g₁₂) := rfl
+
```

Comment:
I guess you might want to consider adding
```lean
@[simps]
def prod_map_monoid_hom : (M →ₗ[R] M) × (M₂ →ₗ[R] M₂) →* ((M × M₂) →ₗ[R] (M × M₂)) :=
```
which I imagine will be useful to you for representation theory